src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -823,6 +829,22 @@ $kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't
     padding: var(--kui-space-40, $kui-space-40);
     position: relative;
 
+    @media (min-width: $kui-breakpoint-laptop) {
+      &:hover{
+        .code-block-secondary-actions{
+          opacity: 1;
+          :deep{
+            .code-block-copy-button{
+              opacity: 1;
+            }
+          }
+          .secondary-actions{
+            opacity: 1;
+          }
+        }
+
+      }
```

Comment:
nit: some formatting issues here; make sure to have spaces before the brackets and empty line between rules (applies to the other changed block as well)

Reply:
Fixed the formatting issues